README.md

```diff
@@ -1,4 +1,4 @@
-# Looping Code-along
+# Looping Lab
```

Comment:
The repo name doesn't / hasn't changed with this change? Does that need to happen? Think thru track impact, etc. Not say9ing it's wrong, just saying it's a thing.

Reply:
Changing the repo name would require altering the track itself, but I would argue that this is not technically a code-along, since we aren't guiding students through to the solution currently or in the proposed wip.

We could do that if that seems more appropriate. Both `for` and `while` loops are introduced in the previous lesson, https://github.com/learn-co-curriculum/js-looping-and-iteration-object-iteration-readme, as well, so I think it would be better to not confuse students by titling this as a code-along in the readme.